Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM gcc:4.9.4
+# assumes this alread has git installed
+#RUN apt-get -y update && apt-get install -y git
+RUN mkdir /opt/externpro
+COPY externpro-16.01.1-gcc494-64-Linux.sh /opt/externpro/
```

Comment:
It has to live in the folder in which the docker is created. I used this to create a docker image and of that up